### ISSUE TITLE
Opening hours

### ISSUE THREAD

**Issue by MichiK** [NoMoKeTo/c3nav#24]
POI opening hours would be a really cool thing to have, especially useful for bars and food courts. "Nearest bar" at 5 am is quite useless if the only bar that is still open at that time is the Uptime Bar.

**Comment by codingcatgirl** [comment]
that is correct, but implementing a full-features opening times data structure would be too much for now.

**Comment by ghost** [comment]
Are there any POIs with different opening times on different days? I don't think so. So opening and closing time would be enough then.
